doc/man3/SSL_CONF_cmd.pod

```diff
@@ -484,6 +493,9 @@ B<EncryptThenMac>: use encrypt-then-mac extension, enabled by
 default. Inverse of B<SSL_OP_NO_ENCRYPT_THEN_MAC>: that is,
 B<-EncryptThenMac> is the same as setting B<SSL_OP_NO_ENCRYPT_THEN_MAC>.
 
+B<IgnoreUnexpectedEOF>: use legacy error handling if the peer does not send
```

Comment:
Using the term `legacy error handling` is wrong.

Reply:
Fixed. Thanks!

Reply:
Fixed. Thanks!